src/table/utils/styling-utils.ts

```diff
@@ -166,6 +171,23 @@ export function getBodyCellStyle(layout: TableLayout, theme: ExtendedTheme): Gen
   };
 }
 
+export const getFooterStyle = (background: BackgroundColors) =>
```

Comment:
i'm not sure about the return type of this function but i think it might help inferring the type while passing into styled components

Reply:
ah yes this should have a type, fixing